src/app/app.component.html

```diff
@@ -87,6 +87,6 @@
     -->
   </ul>
 </div>
-<main class="container-fluid container-pf-nav-pf-vertical">
+<main class="container-fluid container-pf-nav-pf-vertical" *ngIf="loggedIn">
```

Comment:
Yeah I realise that's a bit rubbish and agree we should fix up in future, but works for now.